newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/InAppTutorialPhaseCard.js

```diff
@@ -0,0 +1,173 @@
+// @flow
+
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import Divider from '@material-ui/core/Divider';
+import {
+  ColumnStackLayout,
+  LineStackLayout,
+  ResponsiveLineStackLayout,
+} from '../../../../UI/Layout';
+import Text from '../../../../UI/Text';
+import { type MessageDescriptor } from '../../../../Utils/i18n/MessageDescriptor.flow';
+import { CardWidget } from '../CardWidget';
+import { Column, Line } from '../../../../UI/Grid';
+import ColoredLinearProgress from '../../../../UI/ColoredLinearProgress';
+import Chip from '../../../../UI/Chip';
+import { Trans } from '@lingui/macro';
+import Lock from '../../../../UI/CustomSvgIcons/Lock';
+import GDevelopThemeContext from '../../../../UI/Theme/ThemeContext';
+
+const styles = {
+  cardTextContainer: {
+    flex: 1,
+    display: 'flex',
+    padding: '10px 10px 20px 10px',
+  },
+  image: { height: 130, width: 130 },
+  lockerImage: { height: 80, width: 80 },
+  imageContainer: {
+    height: 130,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  keyPointsList: {
+    paddingInlineStart: 12,
+    textAlign: 'left',
+    overflowWrap: 'anywhere',
+  },
+};
+
+type Props = {|
+  progress?: number,
+  /** For tutorials that cannot be started yet. */
+  locked: boolean,
+  /** To prevent start on click. */
+  disabled?: boolean,
+  title: MessageDescriptor,
+  description: MessageDescriptor,
+  durationInMinutes: number,
+  keyPoints: Array<MessageDescriptor>,
+  onClick: () => void,
+  renderImage: (props: any) => React.Node,
+|};
+
+const InAppTutorialPhaseCard = ({
+  progress,
+  locked,
+  disabled,
+  title,
+  description,
+  durationInMinutes,
+  keyPoints,
+  onClick,
+  renderImage,
+}: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <CardWidget
+          onClick={onClick}
+          size="large"
+          disabled={disabled || locked}
+          shadowed={locked}
+        >
+          <Column noMargin>
+            <div
+              style={{
+                ...styles.imageContainer,
+                backgroundColor: locked
+                  ? gdevelopTheme.paper.backgroundColor.light
+                  : '#7046EC',
```

Comment:
what's that color? should it be in the theme?

Reply:
I'm not sure it deserves to be in the theme, it's one of the Purple variations.